localstack/services/awslambda/lambda_executors.py

```diff
@@ -13,6 +13,7 @@
 from multiprocessing import Process, Queue
 from typing import Dict, Tuple, Union
 
+import localstack.utils.docker
```

Comment:
nitpick: So far, we've usually used relative (not absolute) import names. We could continue using this pattern, e.g. something like: 
```
from localstack.utils import docker as docker_utils
```
(not critical at all, though - shouldn't hold back the merge)

Reply:
Thanks, did not catch that one, pycharm seems to had a hiccup there when moving PortMappings from bootstrap to docker.py